### DISCUSSION TITLE
plugin-ext build error with lint

### DISCUSSION THREAD

**Discussion by yiliang114** [eclipse-theia/theia#11327] [Q&A] [answered]
### Steps to Reproduce:

1. git clone repo
2. yarn
3. build error with lint error 

![image](https://user-images.githubusercontent.com/11473889/175235045-31fa5f3c-260f-4590-bc37-879d30734540.png)
![image](https://user-images.githubusercontent.com/11473889/175235240-f4953d3c-12c1-48be-aa7f-bed30a56b4d8.png)

There was no `lib` folder under `packages/plugin-ext`,  and when i tried to build `plugin-ext` manually,  I got problems of lint. 

![image](https://user-images.githubusercontent.com/11473889/175235943-ac8358fc-5479-41bc-90e6-c91945c4998f.png)

### Additional Information

- Operating System: macos 12.2.1
- Theia Version: @theia/core@1.26.0  @theia/plugin-ext@1.26.0

**Comment by vince-fugnitto** [comment, marked as answer]
@yiliang114 thank you for the issue however I'm not sure I follow.

I tried locally and `master` builds and lints fine, it also does on the latest ci ([build](https://github.com/eclipse-theia/theia/runs/7022727270?check_suite_focus=true), [lint](https://github.com/eclipse-theia/theia/runs/6999266683?check_suite_focus=true)).

> There was no lib folder under packages/plugin-ext, and when i tried to build plugin-ext manually, I got problems of lint.

You mentioned linting **errors** but these are only **warnings** which do not actually fail the build, they just warn about things that should be fixed at a later date like deprecations.
